packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist.res

```diff
@@ -168,6 +168,22 @@ let pdf = (f: float, t: t) => {
 let inv = T.Integral.yToX
 let cdf = T.Integral.xToY
 
+let diff = (arr: array<float>): array<float> =>
```

Comment:
Should go in `E.A` unless there's a module specialized for `array<float>` and we want to keep `E.A` tied to `array<'a>`.